src/homework07/part2_t24_TimeDescription.java

```diff
@@ -0,0 +1,23 @@
+package homework07;
+
+//  Создать класс и объекты описывающие промежуток времени.
+//  Сам промежуток в классе должен задаваться тремя свойствами: секундами, минутами, часами.
+//  Сделать методы для получения полного количества секунд в объекте и сравнения двух объектов (метод должен работать аналогично compareTo в строках).
+//  Создать два конструктора: получающий общее количество секунд, и часы, минуты и секунды по отдельности.
+//  Сделать метод для вывода данных.
+
+public class part2_t24_TimeDescription {
```

Comment:
We don't use snake case for class names. Class name starts from Capital character
Part2T24TimeDescription

Reply:
i forget

Reply:
Thanks